src/pages/index.tsx

```diff
@@ -1,20 +1,41 @@
 import Hero from "@Component/Hero";
+import Slide from "@Component/Slide";
 import DATA from "data/ong";
 import img_index from 'assets/img-index.png';
 
 const HomePage = () => {
   //object of test
-  const data = {
+  const dataHero = {
     title: DATA.name,
     description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labor',
     buttonContent: 'Donate now',
     fillButton: true,
     urlImage: img_index,
   }
```

Comment:
```suggestion


const HomePage = () => {
  //object of test
  const dataHero = {
    title: DATA.name,
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labor',
    buttonContent: 'Donate now',
    fillButton: true,
    urlImage: DATA.img_hero,
  }
```
Consider implement a new field in DATA object to include this new image